CHANGELOG.md

```diff
@@ -64,22 +64,23 @@
 - [Reducing helper methods in `Standard.Base.Meta`.][12031]
 - [Added Table.Offset][12071]
 - [Implemented Generic JDBC connections.][12073]
-- [Added Column.Offset][12092]
+- [Added Column.offset][12092]
```

Comment:
I guess we could fix the Table one as well then?

Reply:
Done